src/style.css

```diff
@@ -1 +1,67 @@
 /* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  --color-active: #00acdc;
```

Comment:
Better to define variables in `:root` [pseudoclass](https://developer.mozilla.org/en-US/docs/Web/CSS/:root)